src/app/app.component.scss

```diff
@@ -242,14 +234,13 @@
 	color: var(--color-purple);
 }
 
-.main-nav-bold{
+.main-nav-bold {
 	font-weight: 700;
 	font-size: 16px;
 	line-height: 30px;
-	color: #FFFFFF;
+	color: #ffffff;
```

Comment:
Why did you make it lowercase? As far as i know hex colors are case insensitive

Reply:
This is what Umut used and I think we need to stick to one either uppercase or lowercase as Hex are not case sensitive.